src/miner/config.tcl.sample.stratum

```diff
@@ -0,0 +1,12 @@
+# pool host address
+set config_host 127.0.0.1
+
+# port 17064 for 'solo' mode and 17065 for 'stratum' mode
```

Comment:
It would be okay if both modes used the same port, since there's no reason for someone to be running both at the same time, and if they really want to they can use `--listen`.

If the ports are going to be different, we can simplify config by having `default_solo_port` and `default_stratum_port` variables here, then choosing the port based on `config_mode`.